Groups/GroupAttendanceMulti.ascx.cs

```diff
@@ -352,6 +365,58 @@ protected void cbAttendee_CheckedChanged( object sender, EventArgs e )
             }
         }
 
+        /// <summary>
+        /// Handles the SelectedIndexChanged event of the ddlAddPersonGroup control.
+        /// </summary>
+        /// <param name="sender">The source of the event.</param>
+        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
+        protected void ddlAddPersonGroup_SelectedIndexChanged( object sender, EventArgs e )
+        {
+            if ( ddlAddPersonGroup.SelectedValue != "" )
+            {
+                ddlAddPersonGroup.Visible = false;
+                ppAddPerson.Visible = true;
+            }
+        }
+
+        /// <summary>
+        /// Handles the SelectPerson event of the ppAddPerson control.
+        /// </summary>
+        /// <param name="sender">The source of the event.</param>
+        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
+        protected void ppAddPerson_SelectPerson( object sender, EventArgs e )
+        {
+            if ( ppAddPerson.PersonId.HasValue )
+            {
+                if ( _attendees != null && !_attendees.Any( a => a.PersonId == ppAddPerson.PersonId.Value ) )
```

Comment:
This could be the issue with adding a person with new code. Need to make sure it is unique group id and personid when not using combined attendance